app/src/main/java/com/bucketlist/ui/main/MainViewModel.kt

```diff
@@ -9,8 +9,8 @@ import androidx.lifecycle.viewModelScope
 import com.bucketlist.dto.BucketListItem
 
 class MainViewModel : ViewModel() {
+    //Use of Array list to get and set bucket list items
     private var _bucketlistitems: MutableLiveData<ArrayList<BucketListItem>> = MutableLiveData<ArrayList<BucketListItem>>()
-
```

Comment:
Lots of unnecessary space in your code, so I deleted a few spaces.

Reply:
The IDE can do this automatically, and you can provide a template for the IDE to use.  Ensure all developers are using this same template, or the one that comes by default with the IDE, and have them reformat before committing.